randovania/data/json_data/prime1.txt

```diff
@@ -2500,12 +2496,29 @@ Asset id: 1681720207
   * Pickup 73; Major Location? False
   > Dock to Quarry Access
       Trivial
+  > Crane Platform
+      Morph Ball and Boost Ball
```

Comment:
Make this path be trivial, since you can access the item with either morph or space jump depending on the path.

Reply:
Ah yeah fair. Perspective threw me off a bit.